community/sync-gateway/2.1.0/README.md

```diff
@@ -86,117 +87,82 @@ $ mv basic-walrus-bucket.json my-sg-config.json
-
-Create a docker network called `couchbase`.
-
-```
-$ docker network create --driver bridge couchbase
-```
-
-Run Couchbase Server in a docker container, and put it in the `couchbase` network.
-
-```
-$ docker run --net=couchbase -d --name couchbase-server -p 8091-8094:8091-8094 -p 11210:11210 couchbase
-```
-
-Now go to the Couchbase Server Admin UI on [http://localhost:8091](http://localhost:8091) (on OSX, replace localhost with the docker machine host IP) and go through the Setup Wizard.  See [Couchbase Server on Dockerhub](https://hub.docker.com/r/couchbase/server/) for more info.
-
-Create a `/tmp/my-sg-config.json` file on your host machine, with the following:
+**Step - 3 :** Create a configuration file as described in the above config section, and customise the server property::
 
 ```
 {
-  "log": ["*"],
+  "logging": {
+    "console": {
+        "enabled": true,
+        "log_level": "info",
+        "log_keys": ["HTTP"]
+    }
+  },
   "databases": {
     "db": {
       "server": "http://couchbase-server:8091",
       "bucket": "default",
+      "username": "Administrator",
+      "password": "password",
       "users": { "GUEST": { "disabled": false, "admin_channels": ["*"] } }
     }
   }
 }
 ```
 
-Start a Sync Gateway container in the `couchbase` network and use the `/tmp/my-sg-config.json` file:
+**Step - 3 :** Start a Sync Gateway container in the `couchbase` network and use the configuration file you just wrote:
 
-```
-$ docker run --net=couchbase -p 4984:4984 -v /tmp:/tmp/config -d couchbase/sync-gateway /tmp/config/my-sg-config.json
-```
+`$ docker run --net=couchbase -p 4984:4984 -v /tmp:/tmp/config -d couchbase/sync-gateway /tmp/config/my-sg-config.json`
 
-Verify that Sync Gateway started by running `docker logs container-id` and trying to run a curl request against it:
 
-```
-$ curl http://localhost:4984
-```
+# Collecting logs via sgcollect_info
 
-## Using sgcollect_info
+This section only applies if you need to run the `sgcollect_info` tool to collect Sync Gateway diagnostics for support.
 
-This section only applies if you need to run the `sgcollect_info` tool to collect Sync Gateway diagnostics for Sync Gateway running in a docker container. In order to collect the logs you will need to do the following workaround:
+**Step - 1 :** Run the following curl command against the admin port of Sync Gateway to run sgcollect_info and put the zip in your log file path.
 
-```
-$ docker logs container-id > /tmp/sync_gateway.log 2>&1
-$ docker exec container-id mkdir -p /var/log/sync_gateway/
-$ docker cp /tmp/sync_gateway.log contaner-id:/var/log/sync_gateway/sync_gateway_error.log
-```
+`# curl -X POST http://localhost:4985/_sgcollect_info -H 'Content-Type: application/json' -d '{"upload": false, "customer": "your-company-name"}'`
 
-Once that is done, you can run `sgcollect_info` via:
+Set upload to `true` along with a customer name if you wish to automatically upload the collection to Couchbase Support.
```

Comment:
```suggestion

```

Tempted to cut this down a bit, and leave it to point to the main docs. Lowers the risk of random uploads/doesn't set the expectation that Support will engage any user.